### DISCUSSION TITLE
193

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#166] [General]
# 潮流周刊第193期 - 猫的陪伴

封面图拍摄于在周天晚上整理潮流周刊的时候，汤圆在书桌的一角安静的呆着，时不时呼呼呼，它很喜欢呆着我在的空间，这种安静的陪伴感甚好。

https://weekly.tw93.fun/posts/193-%E7%8C%AB%E7%9A%84%E9%99%AA%E4%BC%B4

**Comment by xxholly32** [comment]
这个影神图可以啊

**Reply by tw93** [reply]
哈哈 谢谢喜欢

**Comment by VmzXv** [comment]
这周还发吗博主

**Reply by tw93** [reply]
哈哈哈 等下周一加倍

**Comment by ClishWang** [comment]
同问

**Reply by tw93** [reply]
哈哈哈 等下周一加倍

**Comment by duofu** [comment]
B站有黑神话Wiki的，黑神话发售的时候就有了...
